src/plugins/imagery/components/ImageryView.vue

```diff
@@ -774,10 +905,87 @@ export default {
             let index = this.focusedImageIndex;
 
             if (index === this.imageHistory.length - 1) {
-                this.setFocusedImage(this.imageHistory.length - 2, THUMBNAIL_CLICKED);
+                this.thumbnailClicked(this.imageHistory.length - 2);
             } else {
-                this.setFocusedImage(--index, THUMBNAIL_CLICKED);
+                this.thumbnailClicked(--index);
+            }
+        },
+        resetImage(overrideLock) {
+            if (this.panZoomLocked && !overrideLock) {
+                return false;
+            }
+
+            const defaultScale = 1;
+            this.zoomFactor = defaultScale;
+            this.imagePanned = false;
+            this.panZoomLocked = false;
+            this.imageTranslateX = 0;
+            this.imageTranslateY = 0;
+        },
+        zoomImage(newScaleFactor, screenClientX, screenClientY) {
+            if (!this.isPaused) {
```

Comment:
When this method is called as an `emit` from the sub-component, let's ensure that the parameters received are 
`(newScaleFactor, offsetFromCenterX, offsetFromCenterY)`. So any calculations from the `screenClientX` and `screenClientY` can happen in the sub component.